cypress/support/pages/signIn.pageObject.js

```diff
@@ -1,20 +1,29 @@
 import PageObject from '../PageObject';
 
-class SignInPageObject extends PageObject {
-  url = '/#/login';
+class SignUpPageObject extends PageObject {
```

Comment:
The class name `SignUpPageObject` suggests that this file is for sign-up functionality. If this file is intended for sign-in functionality, consider renaming the class and file to reflect that.